.docker/docker-bake.hcl

```diff
@@ -13,7 +13,6 @@ variable "ORGANIZATION" {
 }
 
 variable "REGISTRY" {
-  default = "ghcr.io/"
```

Comment:
Hmm, now I am realizing, that I am not sure whether this change will not break the full build. :thinking:

Reply:
I remember you use `env.hcl` for aiidalab-docker-stack, did you use it here as well?

Reply:
During test, I find it a bit annoying that need to careful about whether put `/`. I think we can always force the `REGISTRY` is set, either to `docker.io` or to `ghcr.io`, then use:

```
 ${REGISTRY:-}/${AIIDA_CORE_WITH_SERVICES_IMAGE:-aiidateam/aiida-core-with-services}:${TAG:-}
```

instead of:

```
 ${REGISTRY:-}${AIIDA_CORE_WITH_SERVICES_IMAGE:-aiidateam/aiida-core-with-services}${TAG:-}
```

Reply:
> Hmm, now I am realizing, that I am not sure whether this change will not break the full build. 🤔

Not sure if it is because of this, but the build _did_ just fail on `main`: https://github.com/aiidateam/aiida-core/actions/runs/9398592676/job/25884242952

Reply:
Seems it is, it tries push to `docker.io` as default registry.